### PR TITLE
Reorganization, test 

### PR DESCRIPTION
**Pull request by jpenzens**

### CODE REVIEW

File: RxODE/RxODE_standard_eval.Rmd [hunk 1]
```diff
@@ -79,52 +81,94 @@ ggplot(infsim, aes(x = time, y = CC)) + geom_line()
...
+* PKPDsim has the best set up for PopPK.
+* PKPDsim and mrgsolve have population parameter variation functionality.  This one does not, you must build outside of R.
+* This is also very convenient set up if we want to specify our own population.  We could also build covariates into our model and specify here too, similarly to mrgsolve.
 
- + 7. Population PK - CV of params: CL 30.13%; V1 22.83%
- + 8. Correlates: CL(ml/min/kg) = 0.67 * CrCl + AGE^.24; V1 invariant
- + 9. Population average CL/Age in study: 61.1(sd = 16.3) y, CL = 86.1(sd = 55.1) mL/min
+#### Loop
+*RxODE is the only thing that requires looping outside of compiled code.
+*lighting fast.  5 mississippi for 10k subjects (i'll test exactly when I have time lol )
```
Comment (dpastoor) [inline RxODE/RxODE_standard_eval.Rmd]: check out the package microbenchmark

you can set up very simple benchmark tests

```
library(microbenchmark)
microbenchmark(func_1_to_test(), func_2_to_test())

```

File: RxODE/RxODE_standard_eval.Rmd [hunk 2]
```diff
@@ -79,52 +81,94 @@ ggplot(infsim, aes(x = time, y = CC)) + geom_line()
...
+```
 
+####Making a function, time testing
+```{r}
 
+popPKsim <- function(thetas, inits, ev, mod){
+  nsub <- as.numeric(nrow(thetas))
+  nobs <- ev$get.nobs()
+  cp.all <- matrix(NA, nobs, nsub)  
+  for (i in 1:nsub){
```
Comment (dpastoor) [inline RxODE/RxODE_standard_eval.Rmd]: we need to 'break' you of your habit of writing loops - its a massive antipattern in R. Though it is fine in python and other languages it is a massive massive performance bottleneck for large numbers of replicates. 

The only time you should use a for loop is for things that you know will never have performance implications (like going through columns in a df or small numbers of reps (like need to run 3 diff scenarios). Anything more you should be using the `apply` class of functions, or wrapping your code in `dplyr::do` calls

File: RxODE/RxODE_standard_eval.Rmd [hunk 3]
```diff
@@ -11,15 +11,15 @@ library(ggplot2)
 library(dplyr)
 ```
 
-####General thoughts after reading viginette, examples, etc
-* **Lets make a standard PopPK test subject number (i say 1000) and time the simulators!!!!**
+####General thoughts 
+* **Lets make a standard PopPK test subject number (i say 10000) and time the simulators!!!!**
```
Comment (dpastoor) [inline RxODE/RxODE_standard_eval.Rmd]: you actually want to time over a range -> 1 subject, 10 subj, 100, 1000, 10000

This is more indicative of various real world scenarios, and also will demonstrate how well the various packages compare with memory management and other things.

**Comment by jpenzens** [comment]
@dpastoor  can you kill this pull request? its already been commited/merged via #10

**Pull request closed without merging**